src/site/content/en/fast/browser-level-image-lazy-loading/index.md

```diff
@@ -218,9 +217,7 @@ Browsers that do not yet support the `loading` attribute will ignore its presenc
 
 ### Are there plans to automatically lazy-load images in Chrome?
 
-Chromium already automatically
-lazy-loads any images that are well suited to being deferred if [Lite
-mode](https://blog.chromium.org/2019/04/data-saver-is-now-lite-mode.html) is enabled on Chrome for Android. This is primarily aimed at users who are conscious about data-savings.
+Previously, Chromium automatically lazy-loaded any images that were well suited to being deferred if [Lite mode](https://blog.chromium.org/2019/04/data-saver-is-now-lite-mode.html) was enabled on Chrome for Android. However, [Lite mode has been deprecated](https://support.google.com/chrome/thread/151853370/sunsetting-chrome-lite-mode-in-m100-and-older?hl=en) and there are currently no plans to provide automatically lazy-load of images in Chrome.
```

Comment:
Ah, ignore my previous comment on Lite Mode/auto-lazy loading. You cover it here :)

Reply:
Think we should add a mention of `auto` in here?

Reply:
Added a comment about that here: https://github.com/GoogleChrome/web.dev/pull/8312/commits/61cb2737393e67ac442e18b31d9416c5c8958e2f